### DISCUSSION TITLE
new vscode option stops keymaps from working

### DISCUSSION THREAD

**Discussion by tedbyron** [LazyVim/LazyVim#803] [General]
It looks like none of my keymaps are working when using the vscode-neovim plugin and the new `{ import = 'lazyvim.plugins.extras.vscode' }`. I was using the vscode extension with lazyvim before, and just disabling packages with `enabled = not vim.g.vscode`, which worked, but after trying switching to lazyvim.plugins.extras.vscode none of my keymaps are applied

**Comment by folke** [comment]
Yes, I'm aware of that. For now just add `{"LazyVim/LazyVim", vscode=true}`

Need to do more testing myself and enable that for everyone by default.

**Reply by tedbyron** [reply]
thanks!

**Reply by folke** [reply]
should be fixed now

**Reply by tedbyron** [reply]
working great, thanks again
